docs/dacce.md

```diff
@@ -0,0 +1,20 @@
+# DACCe
+
+## Usage Example 🚀
```

Comment:
Essas explicações usando o código python poderiam estar agora dentro de uma outra seção chamada "Using in Python Code"

Reply:
Criado mais uma seção para exemplo de codigo Python e excluindo a seção do DACCE